exercises/etl/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+use std::collections::BTreeMap;
+
+pub fn transform(h: &BTreeMap<i32, Vec<char>>) -> BTreeMap<char, i32> {
+    unimplemented!("How will you transform the tree {:?}?", h)
```

Comment:
Please express this in the form of an imperative statement: "Transform this tree: {:?}" or similar.